src/Compiler/Service/FSharpWorkspaceState.fsi

```diff
@@ -0,0 +1,201 @@
+// Copyright (c) Microsoft Corporation.  All Rights Reserved.  See License.txt in the project root for license information.
+
+module FSharp.Compiler.CodeAnalysis.Workspace.FSharpWorkspaceState
```

Comment:
(for something which will likely evolve many times until production, I would prefer to NOT be doing .fsi due to slowdown for contract changes.)